### PR TITLE
Docs: Add description to Expo app

### PR DESCRIPTION
**Pull request by aalises**
#849

### CODE REVIEW

File: apps/mobile/app.json [hunk 1]
```diff
@@ -2,11 +2,14 @@
   "expo": {
     "name": "tequila-client",
     "entryPoint": "./apps/mobile/index.js",
-    "description": "This project is really great.",
+    "description": "Margarita is an Open-Source, Multi-platform travel app for all major platforms — one code for web, iOS and Android. More information on https://margarita.kiwi.com/",
```
Comment (jvaclavik) [inline apps/mobile/app.json]: What about this change?

"Margarita is an open source, multi-platform travel app..."

Comment (aalises) [reply apps/mobile/app.json]: Sure!

File: apps/mobile/app.json [hunk 2]
```diff
@@ -2,11 +2,14 @@
   "expo": {
     "name": "tequila-client",
```
Comment (jvaclavik) [inline apps/mobile/app.json]: Btw I'm also thinking about this name. Why don't we use something like `margarita`?

Comment (aalises) [reply apps/mobile/app.json]: Say no more, maybe something like  `margarita-mobile`?

Comment (jvaclavik) [reply apps/mobile/app.json]: Yes, it's fine. Pls update also url for Expo link in README.md.

Comment (aalises) [reply apps/mobile/app.json]: Done!

**Pull request merged**